packaging/targets/trusty/etc/init/openhim-core.conf

```diff
@@ -0,0 +1,23 @@
+# OpenHIM server upstart config
+
+description "OpenHIM server"
+
+#logs to /var/log/upstart/openhim-core.log
+console log
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+
+
+setuid openhim
+setgid openhim
+
+
+
+
+script
+  chdir /home/openhim/openhim-core-js
+  exec /home/openhim/.nvm/current/bin/node --harmony /home/openhim/openhim-core-js/lib/server.js  --conf=/etc/openhim/config.json
```

Comment:
That should be fine.  I think the only reason this was here was because I was installing under /home/openhim and needed a conf file to get the paths correct.